packages/SystemUI/res-keyguard/values-ru/havoc_strings.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+/**
+ * Copyright (C) 2019 Havoc-OS
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<resources>
+
+    <string name="custom_text_clock_top_text_default">Сейчас</string>
+
+    <string name="text_clock_zero_h_min">Ноль</string>
+    <string name="text_clock_high">Вечер</string>

```

Comment:
just curiosity:
does anyone know what's the difference between `noon` and `high`?

https://github.com/Havoc-OS/android_frameworks_base/blob/76d9e1e290725bebec9a7f045cf9f76d6fc6e807/packages/SystemUI/src/com/android/keyguard/clocks/CustomTextClock.java#L183